internal/operator/operator.go

```diff
@@ -30,10 +31,6 @@ import (
 	"github.com/statnett/image-scanner-operator/internal/resources"
 )
 
-const (
-	ErrCreateCtrl = "unable to create controller"
-)
-
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
```

Comment:
remove this? replace the setupLog ref in Start with logger?

Reply:
Yes, I think it makes less sense to have the contact now.

Thanks, I've removed setupLog completely.